### PR TITLE
FRW-9206 Improvement of artefact storage mechanism.

### PR DESCRIPTION
**Pull request by annakotiuk**
#### Overview

- Ticket: https://spryker.atlassian.net/browse/FRW-9206

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -819,12 +819,10 @@ jobs:
             -   name: Upload artifacts
                 if: failure()
                 run: |
-                    AWS_DEFAULT_REGION=${{env.ROBOT_TESTS_ARTIFACTS_BUCKET_REGION}} AWS_ACCESS_KEY_ID=${{ secrets.ROBOT_TESTS_ARTIFACTS_KEY }} AWS_SECRET_ACCESS_KEY=${{ secrets.ROBOT_TESTS_ARTIFACTS_SECRET }} aws s3 cp .robot/results s3://${{vars.ROBOT_TESTS_ARTIFACTS_BUCKET}}/b2c-robot-api-ci/dynamic-store-off/${GITHUB_RUN_ID}/PHP8.2MariaDBRobot/ \
-                      --recursive \
-                      --expires "$(date -d '+7 days' --utc +'%Y-%m-%dT%H:%M:%SZ')"
+                    AWS_DEFAULT_REGION=${{env.ROBOT_TESTS_ARTIFACTS_BUCKET_REGION}} AWS_ACCESS_KEY_ID=${{ secrets.ROBOT_TESTS_ARTIFACTS_KEY }} AWS_SECRET_ACCESS_KEY=${{ secrets.ROBOT_TESTS_ARTIFACTS_SECRET }} aws s3 cp .robot/results/log.html s3://${{vars.ROBOT_TESTS_ARTIFACTS_BUCKET}}/b2c/dms-off/robot-api/${GITHUB_RUN_ID}/PHP8.2MariaDB/log.html
```
Comment (olhalivitchuk) [inline .github/workflows/ci.yml:822]: So is the `--expires "$(date -d '+7 days' --utc +'%Y-%m-%dT%H:%M:%SZ')"` not needed?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -819,12 +819,10 @@ jobs:
...
 
-    docker-alpine-php-83-postgresql-robot:
-        name: "Docker / Alpine / PHP 8.3 / PostgreSQL / Robot / API"
+    docker-alpine-php-82-postgresql-robot:
```
Comment (olhalivitchuk) [inline .github/workflows/ci.yml:824]: Why was it renamed? SPRYKER_PLATFORM_IMAGE: spryker/php:8.3

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -1005,12 +993,10 @@ jobs:
...
                 if: failure()
                 run: |
-                    AWS_DEFAULT_REGION=${{env.ROBOT_TESTS_ARTIFACTS_BUCKET_REGION}} AWS_ACCESS_KEY_ID=${{ secrets.ROBOT_TESTS_ARTIFACTS_KEY }} AWS_SECRET_ACCESS_KEY=${{ secrets.ROBOT_TESTS_ARTIFACTS_SECRET }} aws s3 cp .cypress s3://${{vars.ROBOT_TESTS_ARTIFACTS_BUCKET}}/cypress-ui/b2c/${GITHUB_RUN_ID}/ \
-                      --recursive \
-                      --expires "$(date -d '+7 days' --utc +'%Y-%m-%dT%H:%M:%SZ')"
+                    AWS_DEFAULT_REGION=${{env.ROBOT_TESTS_ARTIFACTS_BUCKET_REGION}} AWS_ACCESS_KEY_ID=${{ secrets.ROBOT_TESTS_ARTIFACTS_KEY }} AWS_SECRET_ACCESS_KEY=${{ secrets.ROBOT_TESTS_ARTIFACTS_SECRET }} aws s3 cp .cypress s3://${{vars.ROBOT_TESTS_ARTIFACTS_BUCKET}}/b2c/dms-on/cypress/${GITHUB_RUN_ID}/PHP8.3MariaDB/ --recursive
 
-    positive-docker-alpine-php-82-mariadb-robot:
-        name: "[Hackathon] Positive / Docker / Alpine / PHP 8.2 / MariaDB / Robot / API"
+    positive-docker-alpine-php-83-mariadb-robot:
```
Comment (olhalivitchuk) [inline .github/workflows/ci.yml:998]: Same but `SPRYKER_PLATFORM_IMAGE: spryker/php:8.2`

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -1050,12 +1036,10 @@ jobs:
...
                 if: failure()
                 run: |
-                    AWS_DEFAULT_REGION=${{env.ROBOT_TESTS_ARTIFACTS_BUCKET_REGION}} AWS_ACCESS_KEY_ID=${{ secrets.ROBOT_TESTS_ARTIFACTS_KEY }} AWS_SECRET_ACCESS_KEY=${{ secrets.ROBOT_TESTS_ARTIFACTS_SECRET }} aws s3 cp .robot/results s3://${{vars.ROBOT_TESTS_ARTIFACTS_BUCKET}}/b2c-robot-api-ci/${GITHUB_RUN_ID}/PHP8.2MariaDBRobot/ \
-                      --recursive \
-                      --expires "$(date -d '+7 days' --utc +'%Y-%m-%dT%H:%M:%SZ')"
+                    AWS_DEFAULT_REGION=${{env.ROBOT_TESTS_ARTIFACTS_BUCKET_REGION}} AWS_ACCESS_KEY_ID=${{ secrets.ROBOT_TESTS_ARTIFACTS_KEY }} AWS_SECRET_ACCESS_KEY=${{ secrets.ROBOT_TESTS_ARTIFACTS_SECRET }} aws s3 cp .robot/results/log.html s3://${{vars.ROBOT_TESTS_ARTIFACTS_BUCKET}}/b2c/dms-on/robot-api/${GITHUB_RUN_ID}/PHP8.3MariaDB/positive/log.html
 
-    negative-docker-alpine-php-82-mariadb-robot:
-        name: "[Hackathon] Negative / Docker / Alpine / PHP 8.2 / MariaDB / Robot / API"
+    negative-docker-alpine-php-83-mariadb-robot:
```
Comment (olhalivitchuk) [inline .github/workflows/ci.yml:1041]: SPRYKER_PLATFORM_IMAGE: spryker/php:8.2

**Review by olhalivitchuk — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (annakotiuk) [reply .github/workflows/ci.yml:822]: Yes. Expiration date is configured in was

**Review by olhalivitchuk — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 5]
```diff
@@ -1095,6 +1079,4 @@ jobs:
             -   name: Upload artifacts
                 if: failure()
                 run: |
-                    AWS_DEFAULT_REGION=${{env.ROBOT_TESTS_ARTIFACTS_BUCKET_REGION}} AWS_ACCESS_KEY_ID=${{ secrets.ROBOT_TESTS_ARTIFACTS_KEY }} AWS_SECRET_ACCESS_KEY=${{ secrets.ROBOT_TESTS_ARTIFACTS_SECRET }} aws s3 cp .robot/results s3://${{vars.ROBOT_TESTS_ARTIFACTS_BUCKET}}/b2c-robot-api-ci/${GITHUB_RUN_ID}/PHP8.2MariaDBRobot/ \
-                      --recursive \
-                      --expires "$(date -d '+7 days' --utc +'%Y-%m-%dT%H:%M:%SZ')"
+                    AWS_DEFAULT_REGION=${{env.ROBOT_TESTS_ARTIFACTS_BUCKET_REGION}} AWS_ACCESS_KEY_ID=${{ secrets.ROBOT_TESTS_ARTIFACTS_KEY }} AWS_SECRET_ACCESS_KEY=${{ secrets.ROBOT_TESTS_ARTIFACTS_SECRET }} aws s3 cp .robot/results/log.html s3://${{vars.ROBOT_TESTS_ARTIFACTS_BUCKET}}/b2c/dms-on/robot-api/${GITHUB_RUN_ID}/PHP8.3MariaDB/negative/log.html
```
Comment (olhalivitchuk) [inline .github/workflows/ci.yml:1082]:
```suggestion
                    AWS_DEFAULT_REGION=${{env.ROBOT_TESTS_ARTIFACTS_BUCKET_REGION}} AWS_ACCESS_KEY_ID=${{ secrets.ROBOT_TESTS_ARTIFACTS_KEY }} AWS_SECRET_ACCESS_KEY=${{ secrets.ROBOT_TESTS_ARTIFACTS_SECRET }} aws s3 cp .robot/results/log.html s3://${{vars.ROBOT_TESTS_ARTIFACTS_BUCKET}}/b2c/dms-on/robot-api/${GITHUB_RUN_ID}/PHP8.2MariaDB/negative/log.html
```

Comment (olhalivitchuk) [reply .github/workflows/ci.yml:1082]: Please check other places as well

**Review by olhalivitchuk — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 5 above)
Comment (annakotiuk) [reply .github/workflows/ci.yml:1082]: I set php version which is used in deploy file

Comment (olhalivitchuk) [reply .github/workflows/ci.yml:1082]: It has to match the job title

**Review by asmarovydlo — APPROVED** [review]

**Pull request merged**